### PR TITLE
Fix build on cross compilation environment

### PR DESCRIPTION
**Pull request by ribalda**

### CODE REVIEW

**Comment by ypnos** [comment]
Thank you for your contribution. Could you give more detail on what environment it is tha necessitates this fix?

p.s.: I am working on moving over to Qt5. There is only a small issue to be fixed.

**Comment by ribalda** [comment]
Hi

I am preparing an openembedded recipe:

./common/recipes-graphics/gerbil/gerbil_git.bb
ricardo@neopili:~/curro/qt5022/build-qt5022/repo/qtec$ cat
./common/recipes-graphics/gerbil/gerbil_git.bb
DESCRIPTION = "Generic framework for visualization and analysis of
multispectral and hyperspectral data that strives to both bring new
innovations in analysis capabilities and be of use in a wide range of
hyperspectral data applications"
HOMEPAGE = "http://gerbilvis.org/"

LICENSE ="GPLv3"
LIC_FILES_CHKSUM ="file://LICENSE;md5=d32239bcb673463ab874e80d47fae504"

DEPENDS = "boost qt4-x11-free tbb opencv"
RDEPENDS_${PN} += "qt4-x11-free"

SRC_URI = "git://github.com/ribalda/gerbil.git"
SRCREV = "eb92e120f815c0725f11ba0f82cf989376519cd5"

S = "${WORKDIR}/git"

inherit distutils-base cmake qt4x11

export EXTRA_OECMAKE = "\
                        -DQT_MOC_EXECUTABLE=${OE_QMAKE_MOC} \
                        -DQT_UIC_EXECUTABLE=${OE_QMAKE_UIC} \
                        -DQT_RCC_EXECUTABLE=${OE_QMAKE_RCC} \
                        "
do_configure() {

# Ensure we get the cmake configure and not qmake

cmake_do_configure
}

do_install() {
install -d ${D}${bindir}
install -m 755 bin/gerbil ${D}${bindir}/
install -m 755 bin/qgerbil ${D}${bindir}/
}

On Fri, Sep 11, 2015 at 1:20 PM, ypnos notifications@github.com wrote:

> Thank you for your contribution. Could you give more detail on what
> environment it is that makes this fix necessary?
> 
> p.s.: I am working on moving over to Qt5. There is only a small issue to
> be fixed.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/gerbilvis/gerbil/pull/30#issuecomment-139518939.

## 

Ricardo Ribalda

File: CMakeLists.txt [hunk 1]
```diff
@@ -25,7 +25,7 @@ if(MSVC)
 	endif()
 elseif(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
 	# Update if necessary
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long -Wno-reorder -pedantic ")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long -Wno-reorder -pedantic -std=c++1y")
```
Comment (ypnos) [inline CMakeLists.txt:28]: Can you please change to std=c++11 as this is the standard we are working with.

Comment (ribalda) [inline CMakeLists.txt:28]: sure

There is a new pull request

On Fri, Sep 11, 2015 at 1:27 PM, ypnos notifications@github.com wrote:

> In CMakeLists.txt
> https://github.com/gerbilvis/gerbil/pull/30#discussion_r39262355:
> 
> > @@ -25,7 +25,7 @@ if(MSVC)
> >     endif()
> >  elseif(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
> >     # Update if necessary
> > -   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long -Wno-reorder -pedantic ")
> > -   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-long-long -Wno-reorder -pedantic -std=c++1y")
> 
> Can you please change to std=c++11 as this is the standard we are working
> with.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/gerbilvis/gerbil/pull/30/files#r39262355.

## 

Ricardo Ribalda

**Pull request closed without merging**